apps/checker/app/services/EntityHelper.scala

```diff
@@ -0,0 +1,54 @@
+package services
+
+import com.gu.typerighter.model.TextRange
+import opennlp.tools.namefind.{NameFinderME, TokenNameFinderModel}
+import opennlp.tools.tokenize.SimpleTokenizer
+
+import java.io.InputStream
+
+case class EntityInText(word: String, range: TextRange)
+
+class EntityHelper() {
+  val personInputFile: InputStream =
+    getClass.getClassLoader.getResourceAsStream("resources/opennlp/en-ner-person.bin")
+  val personModel = new TokenNameFinderModel(personInputFile)
+  val personFinder = new NameFinderME(personModel)
+
+  val organizationInputFile: InputStream =
+    getClass.getClassLoader.getResourceAsStream("resources/opennlp/en-ner-organization.bin")
+  val organizationModel = new TokenNameFinderModel(organizationInputFile)
+  val organizationFinder = new NameFinderME(organizationModel)
+
+  val locationInputFile: InputStream =
+    getClass.getClassLoader.getResourceAsStream("resources/opennlp/en-ner-location.bin")
+  val locationModel = new TokenNameFinderModel(locationInputFile)
+  val locationFinder = new NameFinderME(locationModel)
```

Comment:
Could the slowness be attributable to this block being run on every check, instead of just once?

Reply:
Ooh – yes. How does it perform when we instantiate the helper once per application?

(I meant to check this, and didn't get to it. Thanks for spotting.)

Reply:
I've made this change: 

https://github.com/guardian/typerighter/pull/443/commits/560ab776c4bcf089b926308da07adffe2d394f7b

Seems promising based off logging some timestamps, construction does indeed take much longer than finding entities! But I haven't yet tested on CODE